primeqa/mrc/run_mrc.py

```diff
@@ -35,43 +33,12 @@
 from primeqa.mrc.processors.preprocessors.tydiqa_google import TyDiQAGooglePreprocessor
 from primeqa.mrc.processors.preprocessors.mrqa import MRQAPreprocessor
 from primeqa.mrc.trainers.mrc import MRCTrainer
-from primeqa.boolqa.run_boolqa_classifier import main as cls_main
+from primeqa.text_classification.run_nway_classifier import main as cls_main
 from primeqa.boolqa.run_score_normalizer import main as sn_main
-
+from run_mrc_utils import object_reference
 from primeqa.tableqa.run_tableqa import run_table_qa
 
-def object_reference(reference_as_str: str) -> object:
```

Comment:
what is all this code that was deleted about?

Reply:
I moved it to [primeqa/mrc/run_mrc_utils.py](https://github.com/primeqa/primeqa/pull/348/files#diff-788079b31f3cd5bdd17cb20c6fe821fc6179726c6f8302bdc2880533cf6324fb) to avoid duplicated code.
its needed by run_nway_classifier.py also.  
I can't import run_mrc into run_nway_classifier because run_mrc imports run_nway_classifier, and this creates circular dependencies.